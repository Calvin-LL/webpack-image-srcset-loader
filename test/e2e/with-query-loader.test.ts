import WISLWebpackTestCompiler from "./helpers/WISLWebpackTestCompiler";

test.each([4, 5] as const)(
  "use with webpack-query-loader",
  async (webpackVersion) => {
    const compiler = new WISLWebpackTestCompiler({ webpackVersion });
    const bundle = await compiler.compile({
      loaderOptions: {
        sizes: ["1x", "300w", "original", "2x"],
      },
      useQueryLoader: true,
    });

    expect(bundle.execute("main.js")).toMatchSnapshot("result");
  }
);
