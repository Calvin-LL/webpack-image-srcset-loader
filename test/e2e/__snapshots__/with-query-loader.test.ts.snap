// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`use with webpack-query-loader: result 1`] = `"fe14ecfa6e9cef9040cc28fed5958c27.jpg 1x, e9bda9871f75495eaf0d6420c8fca85f.jpg 300w, bd2b9fdf768b5eb139702c4c3b4573de.jpg, bd2b9fdf768b5eb139702c4c3b4573de.jpg 2x"`;

exports[`use with webpack-query-loader: result 2`] = `"fe14ecfa6e9cef9040cc28fed5958c27.jpg 1x, e9bda9871f75495eaf0d6420c8fca85f.jpg 300w, bd2b9fdf768b5eb139702c4c3b4573de.jpg, bd2b9fdf768b5eb139702c4c3b4573de.jpg 2x"`;
